File: C_LANG/Colleen.c
#include <stdio.h>
#define QUINE "#include <stdio.h>%1$c#define QUINE %2$c %3$s %2$c"
/* Self replicating */

char*
something(){}

int
main(void)
{
    /* Propagation */
    //printf("%s", QUINE, 10);
    printf(QUINE, 10, 34, QUINE);//, 34);
    something();

    return 0;
}

File: PYTHON/Colleen.py
def something():
    x = 'def something():\n    x = %r\n    print(x %% x)\n\n# Self\n\ndef main():\n# Propagation\n    something()\n\nif __name__ == "__main__":\n    main()'
    print(x % x)

# Self

def main():
# Propagation
    something()

if __name__ == "__main__":
    main()
